system/systemd/journald@netdata.conf

```diff
@@ -0,0 +1,5 @@
+# See journald.conf(5) for details.
+
+[Journal]
+SystemMaxUse=256M
+RuntimeMaxUse=256M
```

Comment:
We almost certainly want the runtime usage limit much lower, as this eats system memory and should only end up coming into play if there is _no_ persistent journal file location. 64MB is probably reasonable.